### PR TITLE
Allowing qemu based images to run without user intervention

### PR DESCRIPTION
**Pull request by rvs**

### CODE REVIEW

File: makeflash.sh [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-#      ./makeflash.sh [-C] <output.img>
+#      ./makeflash.sh [-C size] <output.img>
 #
 MKFLASH_TAG="$(linuxkit pkg show-tag pkg/mkflash)-amd64"
 
 if [ "$1" = "-C" ]; then
-    IMAGE=$2
-    dd if=/dev/zero of=$IMAGE count=$(( 1536 * 1024 )) bs=1024
+    SIZE=$2
+    IMAGE=$3
```
Comment (eriknordmark) [inline makeflash.sh]: Do you want to check you have 3 arguments in this case?

File: makerootfs.sh [hunk 2]
```diff
@@ -3,6 +3,9 @@
 #
 #     ./makerootfs.sh <image.yml> <fs> <output.img>
 #
+# The following env variables change the behaviour of this script
+#     ZEN_DEFAULT_BOOT - sets the default GRUB menu entry
```
Comment (eriknordmark) [inline makerootfs.sh]: Add comment that ZEN_DEFAULT_BOOT=1 means using the qemu-friendly boot?

File: makeflash.sh (hunk 1 above)
Comment (rvs) [reply makeflash.sh]: I guess the question is -- should we really bite the bullet and introduce something like getopts(1) here (or across all of our shell scripts for that matter). For now it feels a bit like an overkill, but let me know if you feel that lacking it makes everything too brittle.

File: makerootfs.sh (hunk 2 above)
Comment (rvs) [reply makerootfs.sh]: Great point! I've added the missing comments

**Pull request merged**